app/views/concerns/entity_displaying_view.rb

```diff
@@ -205,7 +205,9 @@ def entity_death
   end
 
   def entity_date(dates)
-    date_most_accurate(dates)
+    return dates if dates.is_a?(String)
```

Comment:
Since this has now been moved to `date_most_accurate` it can be removed altogether.